internal/provisioner/deployment.go

```diff
@@ -0,0 +1,34 @@
+package provisioner
+
+import (
+	_ "embed"
```

Comment:
Is this used here at all?

Reply:
yep, if I remove it, will get:
```
./embedded.go:8:3: go:embed only allowed in Go files that import "embed"
```

Reply:
I wonder why? I don't see it used anywhere in this file.

Reply:
not sure. but below is from the docs:
```
// Go source files that import "embed" can use the //go:embed directive
// to initialize a variable of type string, []byte, or FS with the contents of
// files read from the package directory or subdirectories at compile time.
```

Reply:
oh, my bad. I thought the comment was about a different file. Here it is not needed. Removed